aerogear/mobile-developer-console
# Agms 711

## Motivation
https://issues.jboss.org/browse/AGMS-711

## What
Re-factor the server updated

## Why
Server event calls increase exponentially with every app added. This may be related to other issues with services not binding to MDC

## How
Add an short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 

## Verification Steps
1. add apps to MDC
2. A reduced number of logs should be seen
3. A services to the apps
4. A reduced numbers of logs should be seen also the `mobile-services.json` should be updated with the new services.

* Try with a high number of apps 100+

## Checklist:


## Progress


## Additional Notes

PS.: Add images and/or .gifs to illustrate what was changed if this pull request modifies the appearance/output of something presented to the users.
